test/Libraries/DynamoPythonTests/PythonEditTests.cs

```diff
@@ -642,7 +641,6 @@ public void VerifySysPathValueForCPythonEngine()
             var pynode = nodeModel as PythonNode;
             UpdatePythonEngineAndRun(pynode, PythonEngineManager.CPython3EngineName);
             sysPathList = GetFlattenedPreviewValues(secondPythonNodeGUID);
-            Assert.AreEqual(sysPathList.Count(), 3);
```

Comment:
We now can't be certain of the exact number of paths. But it should be enough (as we are doing) to check that the custom path is visible in the first node and not in the second.